roles/openshift_master/vars/main.yml

```diff
@@ -7,10 +7,12 @@ openshift_master_policy: "{{ openshift_master_config_dir }}/policy.json"
 scheduler_config:
   kind: Policy
   apiVersion: v1
-  predicates: "{{ openshift_master_scheduler_predicates
+  predicates:
+    - "{{ openshift_master_scheduler_predicates
```

Comment:
without this change, I get master failing with:
```
Feb 07 16:06:53 fedora25 origin-master[6527]: F0207 15:06:53.750381    6536 master.go:276] Unable to start scheduler: invalid scheduler configuration: only encoded map or array can be decoded into a struct
```

and the `scheduler.json` generated is:

```
{
    "apiVersion": "v1", 
    "kind": "Policy", 
    "predicates": {
        "argument": {
            "serviceAffinity": {
                "labels": [
                    "region"
                ]
            }
        }, 
        "name": "Region"
    }, 
    "priorities": {
        "argument": {
            "serviceAntiAffinity": {
                "label": "zone"
            }
        }, 
        "name": "Zone", 
        "weight": 2
    }
}
```

instead, with the patch applied, I get:
```
{
    "apiVersion": "v1", 
    "kind": "Policy", 
    "predicates": [
        {
            "argument": {
                "serviceAffinity": {
                    "labels": [
                        "region"
                    ]
                }
            }, 
            "name": "Region"
        }
    ], 
    "priorities": [
        {
            "argument": {
                "serviceAntiAffinity": {
                    "label": "zone"
                }
            }, 
            "name": "Zone", 
            "weight": 2
        }
    ]
}
```